src/goTest.ts

```diff
@@ -122,7 +122,8 @@ function goTest(config: TestConfig): Thenable<boolean> {
 		outputChannel.show(2);
 		let buildFlags: string[] = vscode.workspace.getConfiguration('go')['buildFlags'];
 		let buildTags: string = vscode.workspace.getConfiguration('go')['buildTags'];
-		let args = ['test', '-v', '-timeout', config.timeout, '-tags', buildTags, ...buildFlags];
+		let testFlags: string[] = vscode.workspace.getConfiguration('go')['testFlags'];
+		let args = ['test', ...testFlags, '-timeout', config.timeout, '-tags', buildTags, ...buildFlags];
```

Comment:
Agreed. Ideally 
- `buildFlags` should be used for build only, 
- `testFlags` should be used for test only.
- `testTimeout` should not be a separate setting and should be included in the `testFlags` setting

If we do this, we won't be backward compatible and everyone who had configured buildFlags and/or testTimeout will have to update their settings to add the new `testFlags` 

@lukehoban Thoughts?